test_util/cluster_api.py

```diff
@@ -244,9 +249,17 @@ def __init__(self, dcos_uri, masters, public_masters, slaves, public_slaves,
                 Note: user must be authenticated explicitly or call self.wait_for_dcos()
             ca_cert_path: (str) optional path point to the CA cert to make requests against
         """
+        # URI must include scheme
```

Comment:
In our case, that URI will _always_ be a URL. That's why I liked calling it a URL :-).

The fact that `URI must include scheme` actually is an artifact of this.